src/common/dto/nonClassMeetings/CreateNonClassParent.dto.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsUUID } from 'class-validator';
+
+export default abstract class CreateNonClassParentDTO {
+  @ApiProperty({
+    type: 'string',
+    example: 'a49edd11-0f2d-4d8f-9096-a4062955a11a',
+    description: 'Area primary key ID',
+  })
+  @IsUUID()
+  public area: string;
+
+  @ApiProperty({
+    description: 'The title of this non class parent',
+    example: 'Reading group',
+  })
+  public title: string;
+
+  @ApiProperty({
+    description: 'The contact name for this non-class parent',
+    example: 'James Waldo',
+    nullable: true,
+  })
+  public contactName?: string;
+
+  @ApiProperty({
+    description: 'The contact email for this non-class parent',
+    example: 'j.waldo@seas.harvard.edu',
+    nullable: true,
+  })
+  public contactEmail?: string;
+
+  @ApiProperty({
+    description: 'The contact phone number for this non-class parent',
+    example: '(123) 456-7890',
+    nullable: true,
+  })
+  public contactPhone?: string;
```

Comment:
As with email above, there is an `@IsPhoneNumber` validator that we can use with `@IsOptional`. It does take a country code as an argument, which we'd want to set to US (though I am _slightly_ concerned that we could have people wanting to use non-US phone numbers here, that seems like an edge case when we're managing physical spaces on a US campus.)

Reply:
Yeah, I think that's somewhat valid but as you mention - it seems extremely unlikely that someone on campus booking a physical space would have a non-US phone number. If this were an app designed for international consumption, perhaps but I agree it seems like a very extreme edge case. If it does come up, we can always loosen the validation to take account of this...

Reply:
Fixed in Fixed in 5ce1671152dbd7199aea9d5a6996bdfe6d161a6e